CONTRIBUTING.md

```diff
@@ -10,7 +10,7 @@ If you would like to contribute enhancements or fixes, please do the following:
 Please note that modifications should follow these coding guidelines:
 
 - Indent is 2 spaces.
-- Code should pass `coffeelint` linter.
+- Code should pass `eslint` linter (run `npm run-script lint`).
```

Comment:
Anything unambiguous would work (like `nmp ru` or `npm run-sc`). `run-script` just happens to be the actual command name, displayed in `npm help`.

Wanna get this changed to the short form?